test/hotreload-test.lua

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env tarantool
+
+local tap = require('tap')
+local ffi = require('ffi')
+
+-- Presence of the methods confirms that correct metatype is set
+-- for the <struct tuple_merge_source> ctype.
+--
+-- Here we don't verify how the methods work.
+local function test_instance_methods_presence(test, tuple_merger)
+    local methods = {
+        'new_buffer_source',
+        'new_table_source',
+        'new',
+        'new_source_fromtable',
+        'new_source_frombuffer',
+        'new_tuple_source',
+    }
+    test:plan(#methods)
+
+    for _, m in ipairs(methods) do
+        test:ok(tuple_merger[m] ~= nil, ('%s is present'):format(m))
+    end
+end
```

Comment:
The comment is about instance methods, but the test verifies module functions. It seems, we need two separate checks for them.

Reply:
Agree. Added.